src/actions/pause/mapFrames.js

```diff
@@ -55,6 +64,63 @@ export function mapDisplayNames(
   });
 }
 
+function isWasmOriginalSourceFrame(frame, getState: () => State): boolean {
+  if (isGeneratedId(frame.location.sourceId)) {
+    return false;
+  }
+  const generatedSource = getSourceFromId(
+    getState(),
+    frame.generatedLocation.sourceId
+  );
+  return generatedSource.isWasm;
+}
```

Comment:
we might want to move this to `utils/frame.js` in the future if it has other use cases.

Reply:
NIT: what do you think of `isOriginalWasmFrame`? We use the term original to refer to other locations without having the source connotation. This might read a bit better...